source/_docs/tools/dev-tools.markdown

```diff
@@ -114,3 +114,15 @@ Event 0 fired 9:53 AM:
     }
 }
 ```
+
+## Statistics
+
+A list of long-term statistic entities will be shown. Some may not be working and will show a **Fix Issue** link. Clicking on this will give a description of the issue and may give you an option to fix the problem.
+
+![Statistics issue message](/images/docs/developer-tools/statistics_issue.png)
+
+Another use of the {% my developer_statistics title="statistics developer tool" %} is to correct any rogue measurements. Click the
```

Comment:
```suggestion
Another use of the {% my developer_statistics title="statistics developer tool" %} is to correct any measurements. Select the
```
Select is usually the preferred term because it is not specific to [mouse interaction](https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/c/click).